test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -607,7 +607,7 @@ PROTO_TOSTRING(IfStatement, statement)
 
```

Comment:
It is worth to explain *why* you changed `else if` to `elseif`.
I propose to tell something like: from the beginning serializer uses keywords `else if` in conditions, however in Lua [^1] these keywords used without whitespace.

I would add this to commit message too:
> fixed elseif solved 863 (2%) errors

[^1]: https://www.lua.org/manual/5.1/manual.html#2.1

Reply:
Fixed.

Reply:
Thanks!